Core/SiteRatingScoreExtension.swift

```diff
@@ -80,6 +80,12 @@ public extension SiteRating {
         if let network = majorTrackerNetworkStore.network(forDomain: domain) { return network.score }
         return 0
     }
+
+    var isMajorTrackerNetwork: Bool {
```

Comment:
I would move this around a bit. Add the logic for working out the `isMajorTrackerNetwork` condition here and then get `isMajorTrackerScore` to use this conditional to work out the score.

OPTIONAL: It would be nice to move the conditionals into the SiteRating itself then we have all the interesting info about a site there and this extension is just about the "score"

Reply:
I don't think that works - the score returned by `isMajorTrackerScore` comes from `network.score` and is used by the site grade directly (divided by 10).

Reply:
Ah yes ignore, I forgot that it was used as a multiplier.